src/linker/WarningCodes.md

```diff
@@ -0,0 +1,74 @@
+# IL Linker Warning Codes 
```

Comment:
Please add a section for error codes as well and link this file from the source code